packages/tasks/src/tasks/text-to-image/spec/input.json

```
@@ -22,7 +22,7 @@
 			"properties": {
 				"guidance_scale": {
 					"type": "number",
-					"description": "For diffusion models. A higher guidance scale value encourages the model to generate images closely linked to the text prompt at the expense of lower image quality."
+					"description": "A higher guidance scale value encourages the model to generate images closely linked to the text prompt at the expense of lower image quality."
```

```suggestion
					"description": "A higher guidance scale value encourages the model to generate images closely linked to the text prompt, but values too high may cause saturation and other artifacts."
```

(If the previous suggestion is accepted)